File: Deberes_PP_Arquitectura-entregable/TI1.3 RESTFUL_JAVA_SINBDD_GR06/02 CLICON/CLICON_RestFull_Java_GR06/src/main/java/ec/edu/monster/cliente/modelo/RespuestaAutenticacion.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.cliente.modelo;


// Este POJO debe coincidir con el del servidor
public class RespuestaAutenticacion {
    private boolean exitoso;
    private String mensaje;
    private String username;

    // Constructor vacío para deserialización
    public RespuestaAutenticacion() {
    }

    // Getters y Setters que coinciden con el servidor
    public boolean isExitoso() { return exitoso; }
    public void setExitoso(boolean exitoso) { this.exitoso = exitoso; }
    public String getMensaje() { return mensaje; }
    public void setMensaje(String mensaje) { this.mensaje = mensaje; }
    public String getUsername() { return username; }
    public void setUsername(String username) { this.username = username; }
}

File: Deberes_PP_Arquitectura-entregable/TI1.4 RESTFUL_DOTNET_SINBDD_GR06/02 CLICON/ConUni_RestFull_DOTNET_GR06/ConUni_RestFull_DOTNET_GR06/ec.edu.monster.modelo/RespuestaAutenticacion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConUni_RestFull_DOTNET_GR06.ec.edu.monster.modelo
{
    /// <summary>
    /// DTO para respuestas de autenticación
    /// </summary>
    public class RespuestaAutenticacion
    {
        public bool Exitoso { get; set; }
        public string Mensaje { get; set; }
        public string Username { get; set; }
    }
}
